chatterbot/storage/sql_storage.py

```diff
@@ -138,23 +151,25 @@ def filter(self, **kwargs):
         all listed attributes and in which all values
         match for all listed attributes will be returned.
         """
-        from chatterbot.ext.sqlalchemy_app.models import Statement, Response
+        from chatterbot.ext.sqlalchemy_app.models import Statement, Response, Tag
 
         session = self.Session()
 
+        tags = kwargs.pop('tags', '')
+
         filter_parameters = kwargs.copy()
 
         statements = []
         # _response_query = None
         _query = None
         if len(filter_parameters) == 0:
-            _response_query = session.query(Statement)
+            _response_query = session.query(Statement).join(Statement.tags).filter(Tag.name.in_(([tags])))
```

Comment:
I'm interested to know, do the extra parenthesis around the `tags` parameter in this line do anything?